build.gradle

```diff
@@ -58,6 +62,8 @@ subprojects {
         }
     }
 
+    project.plugins.apply('net.ltgt.errorprone')
```

Comment:
@vanniktech still playing with it, unfortunately, neither Dagger snapshot nor providing application class helps…